### PR TITLE
Spec

### PR DESCRIPTION
**Pull request by dionyziz**
This is a conversion from Google Docs to markdown so that we can do all development on GitHub and encourage other developers to contribute if they want to.

Please do not report spec typos or mistakes that also exist in the Google Doc in this review; they can be fixed later. Only verify that the markdown conversion from the Google Doc is maintained in the Markdown. If there are typos or other mistakes in the spec that also existed in the Google Doc, please file separate issues on Trello. If there is text missing or other conversion errors that indicate there is a wrong correspondence between the Google Doc and this markdown version, please **do** report them, as we want to spot them before we delete the original Google Doc.

This will allow a swift migration from Google Docs to GitHub and we will be able to get rid of the Google Doc.

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+
+This specification explains the scope of this project and the design details of the application.
+
+ 
+
+It will also be used as a base for the development of the project.
+
+# Engineering team
+
+The project will be implemented by an engineering team led by **Dionysis Zindros**, software engineer at Twitter, and two programmers **Vitalis Salis **and **Dimitris Lamprinos**, software engineering students.
```
Comment (vitsalis) [inline SPECIFICATION.md]: the asterisks appear on the specification on "and" and "Dimitris" but they do not appear on "Vitalis" why?

File: SPECIFICATION.md [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+
+1. For each creature owned by the current player:
+
+    1. If an opponent creature is neighbouring the creature in question, attack any neighbouring opponent creature uniformly at random.
+
+    2. Otherwise, uniformly randomly choose to idle or to move.
+
+    3. If you have chosen to move, then move at a valid diretion uniformly at random.
+
+The strategy illustrates the basic use of the API, including attacking, moving, and idling.
```
Comment (vitsalis) [inline SPECIFICATION.md:648]: Except from the asteriscs I don't see any other problem. ( The asterisks might not be a problem but they seem weird )

Comment (dionyziz) [inline SPECIFICATION.md:648]: Asterisks denote emphasis. See [mark down syntax](http://daringfireball.net/projects/markdown/syntax) and [try rendering the spec](https://github.com/dionyziz/endofcodes/blob/spec/SPECIFICATION.md) to see that everything is fine.

Comment (vitsalis) [inline SPECIFICATION.md:648]: I rendered the spec and I see asterists over the file.
like this paragraph at the engineering team part:
"The project will be implemented by an engineering team led by Dionysis Zindros, software engineer at Twitter, and two programmers Vitalis Salis *_and *_Dimitris Lamprinos, software engineering students."
"Dionyziz Zindros" and "Vitalis Salis *_and *_Dimitris Lamprinos" appear in bold but it still has some asterisks.

Comment (dionyziz) [inline SPECIFICATION.md:648]: Can you check again? I think it's fixed now.

Comment (vitsalis) [inline SPECIFICATION.md:648]: yes it is, thank you :)

**Pull request merged**